components/omega/doc/userGuide/AuxiliaryVariables.md

```diff
@@ -29,3 +29,6 @@ The following auxiliary variables are currently available:
 | VelDel2RelVortVertex | curl of laplacian of horizontal velocity on cells
 | HTracersEdge | thickness-weighted tracers used for fluxes through edges. May be centered, upwinded or a combination of the two
 | Del2TracersCell | laplacian of tracers on cells
+| ZonalStressEdge | zonal component of wind stress on cells
+| MeridStressEdge | meridional component of wind stress on cells
+| NormalStressEdge | normal component of wind stress on edge
```

Comment:
generic comment - do we want this in AuxState?  Would it make sense to have a 'forcing module' where this lives?

Reply:
These are grouped in the `WindForcingAuxVars` class, so they aren't directly members of the aux state class. This is mentioned in the dev docs, but maybe it should also included be in the user docs.